2-mandatory/8-journey-planner.js

```diff
@@ -19,10 +19,16 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
+function journeyPlanner(locations, transportMode) {
+  let arr = [];
+  for (const key in locations) {
+    locations[key].includes(transportMode) ? arr.push(key) : null;
```

Comment:
This works well. You don't need an "else"action of null, this is a bit confusing. using "?  :  " is good when you have an if and an else action, but here you could just use an if statement.